worlds/pokemon_rb/rules.py

```diff
@@ -17,29 +17,23 @@ def prize_rule(i):
         item_rules["Celadon Prize Corner - Item Prize 3"] = prize_rule
 
     if world.accessibility[player] != "locations":
-        world.get_location("Cerulean City - Bicycle Shop", player).always_allow = (lambda state, item:
-                                                                                   item.name == "Bike Voucher"
-                                                                                   and item.player == player)
-        world.get_location("Fuchsia City - Safari Zone Warden", player).always_allow = (lambda state, item:
-                                                                                        item.name == "Gold Teeth" and
-                                                                                        item.player == player)
+        allow_self_locking_items(world.get_location("Cerulean City - Bicycle Shop", player), "Bike Voucher")
+        allow_self_locking_items(world.get_location("Fuchsia City - Safari Zone Warden", player), "Gold Teeth")
```

Comment:
as long as allow_self_locking_items does add_rule(..., "or") it still has to be after all other rules